SingularityUI/app/components/common/Link.cjsx

```diff
@@ -0,0 +1,13 @@
+Link = React.createClass
+
+    getLink: ->
+        return window.config.appRoot + "/task/" + @props.taskId
```

Comment:
Actually, that function isn't even used - it's leftover from when I was doing this a different way, and I guess I forgot to delete it.